lib/core/alice_chopper_response_interceptor.dart

```diff
@@ -101,35 +106,25 @@ class AliceChopperInterceptor
     } catch (exception) {
       AliceUtils.log(exception.toString());
     }
-    return request;
-  }
-
-  /// Handles chopper response and adds data to existing alice http call
-  @override
-  // ignore: strict_raw_type
-  FutureOr<chopper.Response> onResponse(chopper.Response response) {
-    final httpResponse = AliceHttpResponse()..status = response.statusCode;
-    if (response.body == null) {
-      httpResponse
-        ..body = ''
-        ..size = 0;
-    } else {
-      httpResponse
-        ..body = response.body
-        ..size = utf8.encode(response.body.toString()).length;
-    }
 
-    httpResponse.time = DateTime.now();
-    final headers = <String, String>{};
-    response.headers.forEach((header, values) {
-      headers[header] = values;
-    });
-    httpResponse.headers = headers;
+    final Response<BodyType> response = await chain.proceed(chain.request);
 
+    /// Adds data to existing alice http call
     aliceCore.addResponse(
```

Comment:
Not sure but I don't think its needed to add the `Response` afterwards. My guess is the hashcode and this function where used because the previous `request` and `response` callback we're separate. But with the new Interceptors this in not the case any more. So you can proceed the `request` and attach the `response` directly to the same `AliceCall`.

Reply:
I moved the `AliceResponse` to the `AliceCall`, however, I kept the `alice_token` cause I'm not 100% sure if it's needed for any other purpose. Maybe @jhomlala can give some more info here?